src/include/uapi/abi.h

```diff
@@ -51,8 +51,8 @@
 #define __INCLUDE_UAPI_ABI_H__
 
 /** \brief SOF ABI version major, minor and patch numbers */
-#define SOF_ABI_MAJOR 3
```

Comment:
Why? This looks like the same ABI?

Reply:
I suppose you are right. I might've been a tad too reserved on this one.
There is some minor ABI discrepancy between FW and SW though (FW minor version 7, SW is 5), I'll need to have a number pre-assigned to this change :P

Reply:
is there a need for a change at all? If by luck the bitfields are actually used in the same order then we are backwards compatible.
For now the goal is to align on MINOR 6. MINOR 7 and 8 are evolutions that will be tracked as such at the kernel level.

Reply:
Well, kinda, but also not really. It just so happens that with the toolsets we use (xcc & gcc) the change is fully compatible.

Reply:
If binary for the same settings didn't change with that PR there is no need for ABI bump - well, now it's more stable, funny thing is that it would make more sense to bump ABI if you'd just change compiler what will result in different ipcs for the same settings.